site/docs/components/static-list/examples.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+  <LivePreview componentName="static-list" exampleName="DefaultSingle" >
+
+## Default Single
```

Comment:
Don't quite get why we need both single and multiple examples? 
From user perspective, if we can't write some different design guides for both, we may not need both

Reply:
removed single and only kept the multiple example with a Add item to list functionality